Add line_height_factor to canvas (#3217)
beeware/toga: changes/2144.feature.rst

> A line_height argument can now be used to change the line height of multiline text.

Reviewer: ```suggestion
The line height of multiline text on a Canvas can be configured.
```
